tests/manage/monitoring/test_workload_fixture.py

```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf8 -*-
+"""
+Test cases in this file are demonstrating usage of workload storage utilization
+fixtures, and are not expected to be executed in any real test run (hence
+all tests are marked with ``libtest`` marker).
+
+You can execute test cases here to run the workflow implemented in the
+fixtures. Assuming that you are in root directory of ``ocs-ci`` repository and
+that your environment is fully configured, you can do this for example like
+this:
+
+.. code-block:: console
+
+    (venv) [ocsqe@localhost ocs-ci]$ run-ci --cluster-path ~/data/cluster tests/manage/monitoring/test_workload_fixture.py -vvv --pdb
+
+For the purpose of test case automation development, you can also rerun the
+tests using the measurement data from previous test run. To do this, you
+need to create the following ``reuse-workload.yaml`` config file:
+
+.. code-block:: console
+    ---
+    ENV_DATA:
+      measurement_dir: /home/ocsqe/data/measurement_results
```

Comment:
Similarly as above, I would suggest to use some slightly more *generic* path than `/home/ocsqe/` (e.g. as mentioned above, in the [usage](https://github.com/red-hat-storage/ocs-ci/blob/master/docs/usage.md#deployment-of-cluster) doc, there is used `/home/my_user/...`.

Reply:
ok

Reply:
Actually, I can suggest to place the measurement_results into a cluster dir, so that it would be more clear that these data are related to a particular cluster.